File: Javascript/1041.js
var input = `4.5 -2.2`;
let linea1 = input.split(/[\s]/);
let x = linea1[0];
let y = linea1[1];

if (x == 0 && y == 0) {
  console.log("Origem");
} else if (x == 0 && y != 0) {
  console.log("Eixo Y");
} else if (x != 0 && y == 0) {
  console.log("Eixo X");
} else if (x > 0 && y > 0) {
  console.log("Q1");
} else if (x < 0 && y > 0) {
  console.log("Q2");
} else if (x < 0 && y < 0) {
  console.log("Q3");
} else if (x > 0 && y < 0) {
  console.log("Q4");
}

File: Kotlin/1041.kt
import java.util.Scanner

fun main() {
    val input = Scanner(System.`in`);
    val x = input.nextDouble();
    val y = input.nextDouble();
    
    if (x == 0.0 && y == 0.0) {
        println("Origem");
    }
    else if (x == 0.0 && y != 0.0) {
        println("Eixo Y");
    }
    else if (x != 0.0 && y == 0.0) {
        println("Eixo X");
    }
    else if (x > 0.0 && y > 0.0) {
        println("Q1");
    }
    else if (x < 0.0 && y > 0.0) {
        println("Q2");
    }
    else if (x < 0.0 && y < 0.0) {
        println("Q3");
    }
    else if (x > 0.0 && y < 0.0) {
        println("Q4");
    }
}


